src/aggregator/server/http/server.go

```diff
@@ -59,6 +74,20 @@ func (s *server) Serve(l net.Listener) error {
 	mux := http.NewServeMux()
 	registerHandlers(mux, s.aggregator)
 	pprof.RegisterHandler(mux)
+
+	// create and register debug handler
+	debugWriter, err := xdebug.NewZipWriterWithDefaultSources(
```

Comment:
Just to confirm my understanding: in the aggregator we currently have a single HTTP listen address, and no separate debug listen address? I wonder if it's worth splitting off the debug address and having it be opt-in as we do in the dbnode.

Reply:
I actually talked to Rob about this when I implemented it in the coordinator. we decided that it's better to have fewer ports so I just tacked on the debug handler to the listen address.